scripts/lighthouse/index.js

```diff
@@ -0,0 +1,13 @@
+const logResults = require('./logResults');
+
+const config = require(`../../lighthouse`);
+const launchLighthouse = require('./runLighthouse');
```

Comment:
Mixture of ' and ` - consider reformatting (and removing empty line 2?)

Reply:
interesting, eslint creates the extra carriage return if backticks are removed, it's fixed by using single quotes